src/ImageManipulation.php

```diff
@@ -1024,7 +1039,7 @@ public function variantParts($variantName)
             throw new InvalidArgumentException('Invalid variant name arguments: ' . $variantName);
         }
 
-        return array_merge([$matches['format']], $args[0]);
+        return array_merge([$matches['format']], $args);
```

Comment:
See the description of https://github.com/silverstripe/silverstripe-assets/pull/585/commits/a5580a21feac8c8e672b0d19bb8af51d886f4d4a